images/runtime/node/installDependencies.sh

```diff
@@ -20,3 +20,7 @@ fi
 # PM2 is supported as an option when running the app,
 # so we need to make sure it is available in our images.
 npm install -g pm2@3.5.1
+
+# Application-Insights is supported as an option for telemetry when running the app,
+# so we need to make sure it is available in our images.
+npm install -g applicationinsights@1.4.1
```

Comment:
Just for clarification, when is this run? When the images are checked in to the Docker store; aka before exposed to users?

Reply:
yes(kind  of), 
From user prospective: when user run his app on the specific run image, app service pull that specific image from docker store (read mcr), and creates a runtime container which contains this